### PR TITLE
- moving the wordnet package to the ml-common so can be used by both spark and flink

### PR DESCRIPTION
**Pull request by afshinsadeghi**

### CODE REVIEW

File: sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/WordNetSimilarity.scala [hunk 1]
```diff
@@ -6,29 +6,28 @@
...
  *  and ws4j
- *  and ntlk project
+ *  and nltk project
 
  */
-package net.sansa_stack.ml.spark.nlp.wordnet
-
+package net.sansa_stack.ml.common.nlp.wordnet
 
 object WordNetSimilarity extends WordNet {
```
Comment (GezimSejdiu) [inline sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/WordNetSimilarity.scala:14]: Please, could you resolve the Scala-style issues : 
```xml
<file name="/home/gezim/workspace/AfshinML/SANSA-ML/sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/WordNetSimilarity.scala">
<error source="org.scalastyle.file.NewLineAtEofChecker" severity="error" message="File must end with newline character"/>
</file>
```

File: sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/package.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package net.sansa_stack.ml.spark.nlp
+package net.sansa_stack.ml.common.nlp
```
Comment (GezimSejdiu) [inline sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/package.scala:1]: Same here : 
```xml
<file name="/home/gezim/workspace/AfshinML/SANSA-ML/sansa-ml-common/src/main/scala/net/sansa_stack/ml/common/nlp/wordnet/package.scala">
<error source="org.scalastyle.file.NewLineAtEofChecker" severity="error" message="File must end with newline character"/>
</file>
```

File: sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/WUPWordNetSimilarityMeasuresTests.scala [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
-import com.holdenkarau.spark.testing.DataFrameSuiteBase
-import net.sf.extjwnl.data._
+import net.sansa_stack.ml.common.nlp
+import net.sansa_stack.ml.common.nlp.wordnet.{WordNet, WordNetSimilarity}
 import org.scalatest.FunSuite
+//import net.didion.jwnl.data._
+import net.sf.extjwnl.data._
 
-class WUPWordNetSimilarityMeasuresTests extends FunSuite with DataFrameSuiteBase {
+class WUPWordNetSimilarityMeasuresTests extends FunSuite {
```
Comment (GezimSejdiu) [inline sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/WUPWordNetSimilarityMeasuresTests.scala:7]: Same here : 
```xml
<file name="/home/gezim/workspace/AfshinML/SANSA-ML/sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/WUPWordNetSimilarityMeasuresTests.scala">
<error column="0" line="6" source="org.scalastyle.scalariform.SpaceAfterCommentStartChecker" severity="error" message="Insert a space after the start of the comment"/>
<error column="0" line="5" source="org.scalastyle.scalariform.ImportOrderChecker" severity="warning" message="org.scalatest.FunSuite should be in group 3rdParty, not sansa."/>
<error column="0" line="7" source="org.scalastyle.scalariform.ImportOrderChecker" severity="warning" message="net.sf.extjwnl.data._ should be in group 3rdParty, not sansa."/>
</file>
```

File: sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/TestGetMaxDepth.scala [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package net.sansa_stack.ml.common.nlp.wordnet
+
+import net.sansa_stack.ml.common.nlp.wordnet.WordNet
+import net.sf.extjwnl.data._
+import org.scalatest.FunSuite
+
+class TestGetMaxDepth extends FunSuite {
```
Comment (GezimSejdiu) [inline sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/TestGetMaxDepth.scala:6]: Here as well : 
```xml
<file name="/home/gezim/workspace/AfshinML/SANSA-ML/sansa-ml-common/src/test/scala/net/sansa_stack/ml/common/nlp/wordnet/TestGetMaxDepth.scala">
<error column="0" line="4" source="org.scalastyle.scalariform.ImportOrderChecker" severity="warning" message="net.sf.extjwnl.data._ should be in group 3rdParty, not sansa."/>
<error column="0" line="5" source="org.scalastyle.scalariform.ImportOrderChecker" severity="warning" message="org.scalatest.FunSuite should be in group 3rdParty, not sansa."/>
</file>
```

**Review by GezimSejdiu — COMMENTED** [review]
Hi @afshinsadeghi ,
many thanks for your contribution. Could you please have a look and resolve the Scala-style compliant issues (I just mention some of them on the comments).

Best regards,

**Comment by GezimSejdiu** [comment]
Hi @afshinsadeghi ,
any news on the updates?

Best,

**Comment by afshinsadeghi** [comment]
@GezimSejdiu updated

**Comment by GezimSejdiu** [comment]
Thanks a lot @afshinsadeghi . 
um :( still some scala-style complains :) : https://travis-ci.com/SANSA-Stack/SANSA-ML/builds/94345849 
Please, run `mvn install` locally and see if the `scala-style-output.xml` is empty :) 

Best regards,

**Comment by afshinsadeghi** [comment]
@GezimSejdiu I think they are resolved :)

**Pull request merged**